server/vulnerabilities/msrc/analyzer_test.go

```diff
@@ -146,7 +146,7 @@ func TestWinBuildVersionGreaterOrEqual(t *testing.T) {
 			feed:       "10.0.22000.795",
 			os:         "10.0.22621.795",
 			result:     false,
```

Comment:
I believe the logic here is:  `if patched(), then the CVE does not affect this OS`

```suggestion
			result:     true,
```